mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala

```diff
@@ -133,7 +141,10 @@ object MLUtils {
       numFeatures: Int): RDD[LabeledPoint] =
     loadLibSVMFile(sc, path, numFeatures, sc.defaultMinPartitions)
 
-  @deprecated("use method without multiclass argument, which no longer has effect", "1.1.0")
```

Comment:
We still need to keep this as an annotation. `@deprecated` in comments is a JavaDoc tag, while `@deprecated` outside comments is a Scala annotation, which helps generate compiler warnings.

We should keep each PR minimal, so please keep all `@deprecated` annotation unchanged.